install_config/registry/extended_registry_configuration.adoc

```diff
@@ -385,12 +385,78 @@ as well as a risk of serving image layer to user, who is no longer authorized
 to access it.
 ====
 
+[[middleware-cloudfront]]
+==== CloudFront middleware
 The link:https://docs.docker.com/registry/configuration/#cloudfront[*CloudFront*
 middleware extension] can be added to support AWS, CloudFront CDN storage
 provider. CloudFront middleware speeds up distribution of image content
 internationally. The blobs are distributed to several edge locations around the
 world. The client is always directed to the edge with the lowest latency.
 
+[NOTE]
+====
+The link:https://docs.docker.com/registry/configuration/#cloudfront[*CloudFront*
+middleware extension] can be used only together with
+link:https://docs.docker.com/registry/storage-drivers/s3/[S3] storage.
+It is utilized only during blob serving. Therefore, only blob downloads can be
+speeded up, not uploads.
```

Comment:
Consider rewriting last sentence:
Therefore, only blob downloads can be sped up; uploads can not be.